tests/queries/0_stateless/01379_with_fill_several_columns.sql

```diff
@@ -1,21 +0,0 @@
-SELECT 
```

Comment:
You have accidentially deleted our tests.

Reply:
sorry, it is a merge problem, I'll correct that.